go-controller/pkg/ovn/address_set.go

```diff
@@ -357,6 +357,10 @@ func (as *ovnAddressSet) setOrClear() error {
 }
 
 func (as *ovnAddressSet) addIP(ip net.IP) error {
+	if (!config.IPv4Mode && !utilnet.IsIPv6(ip)) ||
```

Comment:
@JacobTanenbaum what if it was put into AddIPs() instead? eg

```
		if utilnet.IsIPv6(ip) && as.ipv6 != nil {
			err = as.ipv6.addIP(ip)
		} else if !utilnet.IsIPv6(ip) && as.ipv4 != nil {
			err = as.ipv4.addIP(ip)
		} else {
			err = fmt.Errorf("blah blah blah")
		}
		if err != nil {
			return err
		}
```

Reply:
done